spec/peeps_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'peeps'
+
+describe Peeps do
+
+  describe '.all' do
+    it 'returns a message from the db' do
+      connection = PG.connect(dbname: 'chitter_test')
+      connection.exec("INSERT INTO peeps (message) values ('This is a peep!');")
+      result = connection.exec('SELECT * FROM peeps;')
+      expect(result[0]['message']).to eq "This is a peep!"
```

Comment:
You've implemented this feature nicely but I think the test could be improved a little because it would still pass even if your query only returned one, of the potentially many, peeps.